packages/react-bootstrap-table2/src/bootstrap-table.js

```diff
@@ -74,7 +76,7 @@ class BootstrapTable extends PropsBaseResolver(Component) {
 
     return (
       <div className="react-bootstrap-table">
-        <table className={ tableClass }>
+        <table id={ id } className={ tableClass }>
```

Comment:
About #235, I think we need to clarify the class/id is on outter div or `table`

Reply:
Sure. Let's discuss further. What I think is that the benefit of customizing `className` or `id` on the table directly is that it not only allows users to pick up whatever they want but also supports `classNames` from `boostrap`. 

For example: [bootstrap4#tables](https://getbootstrap.com/docs/4.0/content/tables/#examples)
> You can also invert the colors—with light text on dark backgrounds—with .table-dark.

```html
<table class="table table-dark">
  { ... }
</table>
```